app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -30,6 +31,8 @@ data class ValidatedIntentData(val url: String, val source: Session.Source)
 object IntentValidator {
     @VisibleForTesting const val DIAL_PARAMS_KEY = "com.amazon.extra.DIAL_PARAM"
     private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
+    private const val IS_QA_BUILD_KEY = "isQA"
```

Comment:
Can you elaborate on the trade-offs of the isQA flag? It seems unnecessary to me: adding any single flag designed for QA is equivalent to adding the `isQA` flag, right?